### DISCUSSION TITLE
Ошибка при компиляции

### DISCUSSION THREAD

**Discussion by Hm7284156** [vvip-68/LedPanelWiFi#3] [Вопросы и ответы] [answered]
C:\Users\Хабиб\Desktop\GyverPanelWiFi-master\GyverPanelWiFi-master\firmware\GyverPanelWiFi_v1.13\e131.ino:393:21: error: cannot convert 'CRGB' to 'uint32_t' {aka 'unsigned int'} in assignment
C:\Users\Хабиб\Desktop\GyverPanelWiFi-master\GyverPanelWiFi-master\firmware\GyverPanelWiFi_v1.13\e131.ino:463:21: error: cannot convert 'CRGB' to 'uint32_t' {aka 'unsigned int'} in assignment

exit status 1

Compilation error: cannot convert 'CRGB' to 'uint32_t' {aka 'unsigned int'} in assignment

**Comment by vvip-68** [comment, marked as answer]
Неверная версия ядра, неверная версия библиотек, неверная версия IDE...
В общем - использование не рекомендованных версий всего, а чего-то случайного...

**Reply by Hm7284156** [reply]
Хорошо
